timeseries/src/autogluon/timeseries/dataset/ts_dataframe.py

```diff
@@ -96,15 +96,20 @@ def _constructor(self) -> Type[TimeSeriesDataFrame]:
 
     @property
     def freq(self):
-        ts_index = self.index.levels[1]  # noqa
+        ts_index = self.index.get_level_values(1)  # noqa
```

Comment:
Do we ever even use `ts_index` to infer the frequency here? 

I thought that `self.index` is always a `pd.MultiIndex` where the first level is the item ID and the second level are the timestamps for each individual time series. Therefore, `ts_index = self.index.get_level_values(1)` will always contain the concatenated timestamps for all individual time series (most likely not even sorted chronologically) where `freq` and `inferred_freq` don't make that much sense.